resources/authentication_test/X509/ca.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should this whole `resources` directory be moved into either `src/integration_test` or `src/test`?  Located at the top-level like this, it's unclear whether it will be included in the production jar file (not claiming that it is, but the placement makes you wonder).

Reply:
I checked the output for `gradlew shadowJar` which outputs our uberjar and it doesn't contain the contents of this `resources/` directory.  We put our utility scripts and some test files in this directory.  I can create a ticket if this should be moved, for now it seems to be okay.